tests/_data/dummy_projects/with-lockfile/node_modules/my-local-a/LICENSE

```diff
@@ -0,0 +1 @@
+Expecting, that this content will appaear base-64 encoded in the SBoM, if you work with parameter --add-license-text
```

Comment:
:x: I disagree with the abuse of this setup for yet another test. 
This setup's purpose is for a total other setup and there is actually nothing for your use case here.

Reply:
See above